org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolutionAcceptor.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2020 TypeFox GmbH (http://www.typefox.io) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.ide.editor.quickfix;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.xtext.ide.serializer.IChangeSerializer.IModification;
+import org.eclipse.xtext.ide.server.codeActions.ICodeActionService2.Options;
+
+import com.google.common.collect.Lists;
```

Comment:
unused import

Reply:
Thanks. I noticed a lot of .project and .classpath files. Do you generally gitignore eclipse prefs? Can I add/enhance save actions? I'll do an organize imports over my files anyway.

Reply:
nope. they should be checked in. 
but as we are in transition to Java 11 such
discrepancies as yours may happen unfortunately